cli/src/commands/branch.rs

```diff
@@ -122,6 +124,21 @@ pub struct BranchForgetArgs {
     pub glob: Vec<StringPattern>,
 }
 
+/// Rename `old` branch name to `new` branch name.
+///
+/// The new branch name points at the same commit as the old
+/// branch name.
+#[derive(clap::Args, Clone, Debug)]
+pub struct BranchRenameArgs {
+    /// The old name of the branch.
+    #[arg(required = true)]
```

Comment:
nit: `#[arg(required = true)]` can be removed as it is the default for positional arguments.

Reply:
Nice. Removed.